File: simple-fun-number-184-lcm-from-m-to-n/simple-fun-number-184-lcm-from-m-to-n.py
import functools
def gcd_rec(a, b): 
    if b:
        return gcd_rec(b, a % b)
    else:
        return a

def mn_lcm(m,n):
  return functools.reduce(lambda a,b : a*b/gcd_rec(a,b), range(min(n,m),max(n,m)+1) ) 

File: simple-fun-number-184-lcm-from-m-to-n/simple-fun-number-184-lcm-from-m-to-n.js
function gcd_rec(a, b) {
    if(b){return gcd_rec(b, a % b)}
    else{return Math.abs(a)}}

function mnLCM(m,n) {
  var lst = [];
for (var i = Math.min(n,m); i <= Math.max(n,m); i++){lst.push(i)}
console.log(lst)
  return lst.reduce( function(a,b){return a*b/gcd_rec(a,b) } ) 
}